fail_level.go

```diff
@@ -0,0 +1,83 @@
+package reviewdog
+
+import (
+	"fmt"
+
+	"github.com/reviewdog/reviewdog/proto/rdf"
+)
+
+// FailLevel represents enumeration of available filter modes
+type FailLevel int
+
+const (
+	// FailLevelDefault represents default mode, which means users doesn't specify
+	// fail-level. Basically, it's same as FailLevelNone.
+	FailLevelDefault FailLevel = iota
+	FailLevelNone
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported const FailLevelNone should have comment (or a comment on this block) or be unexported
<!-- __reviewdog__:ChA5ZTMzYjAyMzU2OTc1MzQ0Eg9nb2xpbnQtY2lyY2xlY2k= -->